install.sh

```diff
@@ -182,8 +182,8 @@ case "${ARCH}" in
   sha256s+=('46241ea5cf5e0181649e91edb183128669d5ab5bc13a938493ba1158a5189743')
   urls+=('https://dl.bintray.com/chromebrew/chromebrew/libiconv-1.16-3-chromeos-x86_64.tar.xz')
   sha256s+=('52e602f63df955c451b36ff0b204331ed5c336d384f6484b2f7d5ef7a43f2a6d')
-  urls+=('https://dl.bintray.com/chromebrew/chromebrew/libidn2-2.0.5-chromeos-x86_64.tar.xz')
-  sha256s+=('22b43990ca6ae415b9b7e206ed7d7b05df29b97411de55f33f11e834cf3b345c')
+  urls+=('https://dl.bintray.com/chromebrew/chromebrew/libidn2-2.3.0-chromeos-x86_64.tar.xz')
+  sha256s+=('06c9f3803f02ce792550a39a1de3253b74ca6622b2543f843727f31dc4aa0f46')
```

Comment:
I believe this is a prerequisite dependency for git.  It doesn't break the current version on a fresh install, correct?  FYI, any time this file is updated, fresh install testing is mandatory.

Reply:
Should I add the updated git and check that together?

Reply:
Only if something breaks.  It's best to update incrementally.

Reply:
Reworked install.sh attach to this PR with newer libidn2 & git boots fine in a armv7l vm.

Reply:
Let me know if you need any changes, otherwise this is good to go?

(Also the install.sh attached to this already has the updates for git in https://github.com/skycocker/chromebrew/pull/4829 )